### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by amitiuttarwar**

### CODE REVIEW

File: _posts/2020-10-07-#19843.md [hunk 1]
```diff
@@ -10,14 +10,51 @@ status: upcoming
...
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* The Bitcoin network uses `addr` messages to communicate addresses aka
+  locations of nodes. To read more about the P2P message, you can see these
+  [docs](https://en.bitcoin.it/wiki/Protocol_documentation#addr).
```
Comment (jonatack) [inline _posts/2020-10-07-#19843.md:17]: maybe:

"The Bitcoin P2P network uses `addr` messages to communicate addresses, aka the locations of nodes. To read more about the `addr` message, see these"...

**Review by jonatack — COMMENTED** [review]
Looking like an excellent session. Thanks @amitiuttarwar!

File: _posts/2020-10-07-#19843.md (hunk 1 above)
Comment (jnewbery) [reply _posts/2020-10-07-#19843.md:17]: Updated. Thanks Jon!

**Comment by jnewbery** [comment]
Thanks for the notes and questions @amitiuttarwar! Looking forward to the meeting next week :)

**Pull request merged**